lab_6_pipeline/pipeline.py

```diff
@@ -165,22 +308,77 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        super().__init__(corpus_manager)
+        self._backup_analyzer = pymorphy2.MorphAnalyzer()
+        mapping_path = Path(__file__).parent / 'data' / 'opencorpora_tags_mapping.json'
+        self._backup_tag_converter = OpenCorporaTagConverter(mapping_path)
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        sentences = []
+        result = self._stemmer.analyze(re.sub(r'\W+', ' ', text))
+        token_count = 0
+        for sentence_position, sentence in enumerate(split_by_sentence(text)):
```

Comment:
also think about large duplication between pipelines - how could you manage it? remember, API cannot be changed

Reply:
fixed